src/inspector/view/qml/MuseScore/Inspector/popups/text/TextStylePopup.qml

```diff
@@ -0,0 +1,472 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2023 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import MuseScore.Ui 1.0
+import MuseScore.UiComponents 1.0
+import MuseScore.Inspector 1.0
+
+import "../../common"
+
+StyledPopupView {
+    id: root
+
+    property QtObject model: textStyleModel
+
+    // The navigation order does not match the order of the components in the file
```

Comment:
To make clear that this comment is there for documentation and not as a ToDo, you could do this, like we often do with comments like that:
```suggestion
    //! Note: the navigation order does not match the order of the components in the file
```